iosApp/iosApp/Utils/AttributedStringExtension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  AttributedStringExtension.swift
+//  iosApp
+//
+//  Created by Kayla Brady on 4/14/25.
+//  Copyright © 2025 MBTA. All rights reserved.
+//
+
+import SwiftUI
+
+extension AttributedString {
+    static func boldOrDefault(_ stringWithMarkdown: String) -> AttributedString {
```

Comment:
I find this function name confusing: it doesn’t make any text bold, it just renders Markdown that may call for some text to be bold. `tryMarkdown` might make more sense.

Reply:
I like that way better, TY!